templates/gatsby-wordpress/files/frontend/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this an alternate editorconfig or something?  I presume it just comes with gatsby?

Reply:
Yeah, all the "pretty" stuff is gatsby generated.